src/components/app/AuthenticatedPageContext.jsx

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export default createContext();
```

Comment:
[inform] This context is defined in its own standalone file instead of within `AuthenticatedPage` in order to avoid a import dependency cycle.